### PR TITLE
CIV-6918 fix for caselocation

### PR DESCRIPTION
**Pull request by drummondjm**
fix for case location,
we were previously  implementing a "predefined" type, this removes that.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by maciejmodzelewski — APPROVED** [review]

**Review by sankhajuria — DISMISSED** [review]

**Review by sampankumar — DISMISSED** [review]

File: charts/civil-ccd/values.preview.template.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 civil-service:
   java:
+    image: 'hmctspublic.azurecr.io/civil/service:pr-2217'
```
Comment (mounikahmcts) [inline charts/civil-ccd/values.preview.template.yaml:3]: is it expected?

**Review by sankhajuria — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]

File: charts/civil-ccd/values.preview.template.yaml (hunk 1 above)
Comment (sankhajuria) [reply charts/civil-ccd/values.preview.template.yaml:3]: We will remove this once civil service  merged with master

File: ccd-definition/ComplexTypes/CaseLocationDJ.json [hunk 2]
```diff
@@ -1,20 +1,20 @@
 [
 
   {
-    "ID": "CaseLocation",
+    "ID": "CaseLocationCivil",
     "CaseEventID": "DEFAULT_JUDGEMENT",
     "CaseFieldID": "region",
     "ListElementCode": "region",
-    "ElementLabel": " ",
+    "ElementLabel": "Reason",
```
Comment (HarryH96) [inline ccd-definition/ComplexTypes/CaseLocationDJ.json:8]: Reason? Should this not be Region?

Comment (sankhajuria) [reply ccd-definition/ComplexTypes/CaseLocationDJ.json:8]: yes, it should be Region, we will fix it. Thanks

**Review by sankhajuria — DISMISSED** [review]

**Review by mounikahmcts — APPROVED** [review]

**Review by sankhajuria — APPROVED** [review]

**Pull request merged**